webpack/assets/javascripts/foreman_users.js

```diff
@@ -9,3 +10,24 @@ export function initInheritedRoles() {
              .html(`${_.escape(target.text)} <span class="caret"></span>`);
   }).first().click();
 }
+
+/* eslint-disable no-undef */
```

Comment:
need to disable no-undef here because `notify` has not been migrated to the new code yet.